fili-core/src/test/groovy/com/yahoo/bard/webservice/table/MetricUnionCompositeTableDefinitionSpec.groovy

```diff
@@ -33,4 +38,85 @@ class MetricUnionCompositeTableDefinitionSpec extends Specification {
         expect:
         metricUnionCompositeTableDefinition.getPhysicalTables(resourceDictionaries) == [physicalTable] as Set
     }
+
+    def "test if getAvailabilitiesToMetrics maps availabilities to metrics"() {
```

Comment:
Can we try to make this test shorter, if not I am ok with it. Just make sure @michael-mclawhorn  takes a look next week.